pkg/kubeconfig/loader.go

```diff
@@ -17,7 +17,14 @@ func LoadKubeconfig(path string) (*rest.Config, error) {
 		loadingRules.ExplicitPath = path
 	}
 	loader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, &clientcmd.ConfigOverrides{})
-	return loader.ClientConfig()
+	config, err := loader.ClientConfig()
+	if err != nil {
+		return nil, err
+	}
+	config.UserAgent = "opencost"
+	config.AcceptContentTypes = "application/vnd.kubernetes.protobuf,application/json"
+	config.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
https://kubernetes.io/docs/reference/using-api/api-concepts/#alternate-representations-of-resources

Reply:
Very very cool! I wonder why they haven't switched to this by default. Backwards compatibility I guess?

Reply:
@r2k1 Just want to make sure that older versions of kubernetes will still work appropriately with this change. Is there any reason to expect versions of kubernetes on GKS, EKS, AKS may not work properly? 

Just want to make sure that we're covering our ground here.

Reply:
I'm not aware about any potential reasons why it would not work. I would expect all cloud services to use the same binary.

kube-state-metrics sets same values.